build_tools/docker/dockerfiles/mmperf.Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The IREE Authors
+# Copyright 2022 The IREE Authors
```

Comment:
I think we don't need to update the date on the license here.

Reply:
Yeah so the full story on copyright years is https://opensource.google/documentation/reference/copyright. The relevant section is:

> the year the file was created in full numeric form (e.g., 2010); don't change this if you edit, move, or copy the file.

So if some file is a "branch" of another, then you leave the old copyright year, but if you create a new file then you put the current year. There's not a hard boundary between those and it's a judgement call. Like if you copied base.Dockerfile and then made some edits, it would make sense to leave the old year, but if you just copy the license header, then update it.

Armed with this information, do whatever you think makes sense.

Reply:
It makes more sense to make it the current year since this is a new file.